lib/index.d.ts

```diff
@@ -57,7 +57,7 @@ export interface ClientOptions {
   queueDisable?: boolean;
   bindDN?: string;
   bindCredentials?: string;
-  createConnection?: (port: number, host: string, onConnect: () => void, onSocketDefined: (socket: net.Socket) => void);
+  createConnection?: (port: number, host: string, onConnect: () => void, onSocketDefined: (socket: net.Socket) => void, onError: (error: any, result: any) => void);
```

Comment:
What's the reason for allowing the `onError` callback to also pass a `result` rather than just limiting it to passing an `error`?

Reply:
I guess it doesn't need the result, so I'll remove it.